priv/repo/update_repo_data.exs

```diff
@@ -39,6 +39,8 @@ defmodule Adoptoposs.UpdateReposityData do
     project = Submissions.get_project!(project_id)
 
     if project.repo_id == repository.id do
+      IO.puts("updating project #{project.id} (repo #{repository.id})")
```

Comment:
We might want to `Logger` instead?

Reply:
Thanks, changed it to use Logger instead of IO.